dts/arm/st/h7/stm32h723.dtsi

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2020 Alexander Kozhinov
+ * AlexanderKozhinov@yandex.com
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/h7/stm32h7.dtsi>
+
+/ {
+	/* DTCM memory directly coppled to CPU */
+	dtcm: memory@20000000 {
+		compatible = "arm,dtcm";
+		reg = <0x20000000 DT_SIZE_K(128)>;
+	};
+
+	/* AXI SRAM in D1 domain (AXI bus) */
+	sram0: memory@24000000 {
+		reg = <0x24000000 DT_SIZE_K(128)>;
+		compatible = "mmio-sram";
+	};
+
+	/* RAM shared between ITCM and AXI in D1 (AXI bus) */
+	sram1: memory@24020000 {
+		reg = <0x24020000 DT_SIZE_K(192)>;
+		compatible = "mmio-sram";
+	};
+
+	/* SRAM1 in D2 domain (AHB bus) */
+	sram2: memory@30000000 {
+		reg = <0x30000000 DT_SIZE_K(16)>;
+		compatible = "mmio-sram";
+	};
+
+	/* SRAM2 in D2 domain (AHB bus) */
+	sram3: memory@30004000 {
+		reg = <0x30004000 DT_SIZE_K(16)>;
+		compatible = "mmio-sram";
+	};
+
+	/* SRAM4 in D3 domain (AHB bus) */
+	sram4: memory@38000000 {
+		reg = <0x38000000 DT_SIZE_K(16)>;
+		compatible = "mmio-sram";
+	};
+
+	/* Backup SRAM1 in D3 domain (AHB bus) */
+	sram5: memory@38800000 {
```

Comment:
I'd use another node label (to emphasize it's a backup SRAM). Datasheet has no references to SRAM5. There should also be some facilities to use these memories (probably add support in the linker scripts for these regions).

Reply:
Thank you. Changed to: backup_sram